javascript-carousel/main.js

```diff
@@ -0,0 +1,69 @@
+var $next = document.querySelector('.next');
+var $previous = document.querySelector('.previous');
+var $currentImage = document.querySelector('.first');
+var $dotContainer = document.querySelector('.dots');
+
+var clickCounter = 0;
+var clickBack = 0;
+
+$next.addEventListener('click', function (event) {
+  clickCounter++;
+  if (clickCounter === 1) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else if (clickCounter === 2) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (clickCounter === 3) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (clickCounter === 4) {
+    $currentImage.setAttribute('src', './images/039.png');
+  } else {
+    $currentImage.setAttribute('src', './images/001.png');
+  }
+  changeImage();
+});
+
+$previous.addEventListener('click', function (event) {
+  clickBack++;
+  if (clickBack === 1) {
+    $currentImage.setAttribute('src', './images/039.png');
+  } else if (clickBack === 2) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (clickBack === 3) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (clickBack === 4) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else {
+    $currentImage.setAttribute('src', './images/001.png');
+  }
+});
+
+
+$dotContainer.addEventListener('click', (event) => {
+  if (event.target && event.target.matches('input.dot1')) {
+    $currentImage.setAttribute('src', './images/001.png');
+  } else if (event.target && event.target.matches('input.dot2')) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else if (event.target && event.target.matches('input.dot3')) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (event.target && event.target.matches('input.dot4')) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (event.target && event.target.matches('input.dot5')) {
+    $currentImage.setAttribute('src', './images/039.png');
+  }
```

Comment:
In here, try looping through all of the dots in a NodeList (see the `javascript-view-swapping` exercise).

Reply:
![Screen Shot 2021-03-12 at 9 51 14 AM](https://user-images.githubusercontent.com/74885918/110978658-83e1d000-8318-11eb-8e2a-b9fe83950602.png)